app/assets/stylesheets/govuk_publishing_components/components/_summary-card.scss

```diff
@@ -0,0 +1,16 @@
+@import "govuk_publishing_components/individual_component_support";
+@import "govuk/components/summary-list/summary-list";
+
+.gem-c-summary-card {
+  .govuk-summary-list__key {
+    vertical-align: top;
+  }
+
+  img {
+    display: block;
+  }
+
+  .govuk-summary-card__action:not(:first-child):not(:last-child) {
```

Comment:
A small suggestion - I wonder if nesting the pseudo selectors might be a little more readable?
```
  .govuk-summary-card__action {
    &:not(:first-child):not(:last-child) {
      padding-left: govuk-spacing(2);
    }
  }
```